util/perl/OpenSSL/Test.pm

```diff
@@ -330,6 +330,25 @@ sub app {
     }
 }
 
+sub fuzz_tests {
+    my @fuzzers = @_;
+
+    plan tests => scalar @fuzzers;
+
+    foreach my $f (@fuzzers) {
+        subtest "Fuzzing $f" => sub {
+            my @dir = glob(srctop_file('fuzz', 'corpora', "$f"));
+
+            plan skip_all => "No directory fuzz/corpora/$f" unless @dir;
+            plan tests => scalar @dir; # likely 1
+
+            foreach (@dir) {
+                ok(run(fuzz(["$f-test", $_])));
+            }
+        }
+    }
+}
+
```

Comment:
This function is not locate where I would expect.  I frankly don't see any viable reason why it shouldn't stay in `99-test_fuzz.t`, but if you must, I would much rather see it in `util/perl/OpenSSL/Test/Utils.pm`

Reply:
The motivation to have `fuzz_tests` in a lib was that this way fuzzers could be partitioned easily,
splitting the calls of the resulting partitions of sub-tests into several `recipes/*.t` files.
So I've just moved it to the other lib source file, `Test/Utils.pm`.

Reply:
Sorry for being picky, but I'm rethinking this...  it doesn't feel like such a generic utility, but rather specialized.  So following what we do with `tconversion`, let me suggest that this function end up in, say, `test/recipes/fuzz.pl`...  don't forget to end that module with a `1;` (yes, after everything!), and then have this in the appropriate recipes:

``` perl
require_ok(srctop_file('test','recipes','fuzz.pl'));
```

Reply:
`require_ok` counts as a test in itself, so remember to increment the plan by 1

Reply:
All right, done.
On this occasion I also considered moving the fuzz-related Perl code from `Test.pm` to the new `fuzz.pl`,
but it looks like that code is strongly related to the rest of `Test.pm`.